routes/web.php

```diff
@@ -11,10 +11,41 @@
 |
 */
 
+// Admin (Service) Authentication Routes...
+Route::get('login', [
+    'as' => 'service.login',
+    'uses' => 'Service\Auth\LoginController@showLoginForm',
+]);
+
+Route::post('login', 'Service\Auth\LoginController@login');
+
+Route::post('logout', [
+    'as' => 'service.logout',
+    'uses' => 'Service\Auth\LoginController@logout',
+]);
+
+// Admin (Service) Password Reset Routes...
+Route::get('password/reset', 'Service\Auth\ForgotPasswordController@showLinkRequestForm')
+    ->name('password.request')
+;
+Route::post('password/email', 'Service\Auth\ForgotPasswordController@sendResetLinkEmail')
+    ->name('password.email')
+;
+Route::get('password/reset/{token}', 'Service\Auth\ResetPasswordController@showResetForm')
+    ->name('password.reset')
+;
+Route::post('password/reset', 'Service\Auth\ResetPasswordController@reset');
+
 // Service Dashboard route
-Route::get('/dashboard', 'Service\DashboardController@index')->name('service.dashboard');
+// Default redirect to Service Dashboard
+Route::get('/', function () {
+    return redirect()->route('service.dashboard');
+})->name('service.base');
```

Comment:
Why service.base and service.dashboard?

Reply:
Should this be in the auth middleware group? The dashboard isn't public.

Reply:
Or is there something else going on here that I am not seeing?

Reply:
service.base is an alias for /
service.dashboard is the first useful landing page.
It seemed logical to redirect / requests to dashboard
the dashboard is protected by auth logic, you can't get there if you arn't auth'd

Reply:
ah.